### PR TITLE
Migrate to JUnit5

### PR DESCRIPTION
**Pull request by majst01**
I would like to switch to the most recent version of junit, this is the first shot.

@fmachado would you please review ?

### CODE REVIEW

**Comment by majst01** [comment]
Code Coverage change far too much, which i don't understand. Maybe junit5 generates different coverage files ?

File: src/test/java/org/influxdb/InfluxDBTest.java [hunk 1]
```diff
@@ -323,10 +326,10 @@ public void testWriteMultipleStringDataLinesThroughUDP() {
         Query query = new Query("SELECT * FROM " + measurement + " GROUP BY *", UDP_DATABASE);
         QueryResult result = this.influxDB.query(query);
 
-        Assert.assertEquals(3, result.getResults().get(0).getSeries().size());
-        Assert.assertEquals(result.getResults().get(0).getSeries().get(0).getTags().get("atag"), "test1");
-        Assert.assertEquals(result.getResults().get(0).getSeries().get(1).getTags().get("atag"), "test2");
-        Assert.assertEquals(result.getResults().get(0).getSeries().get(2).getTags().get("atag"), "test3");
+        Assertions.assertEquals(3, result.getResults().get(0).getSeries().size());
+        Assertions.assertEquals(result.getResults().get(0).getSeries().get(0).getTags().get("atag"), "test1");
```
Comment (fmachado) [inline src/test/java/org/influxdb/InfluxDBTest.java]: (minor adjustment) the "expected value" should be the first parameter

**Comment by fmachado** [comment]
Codecov report seems to be correct now (it was updated 1h ago).

File: src/test/java/org/influxdb/InfluxDBTest.java (hunk 1 above)
Comment (majst01) [reply src/test/java/org/influxdb/InfluxDBTest.java]: good  catch will fix.

**Comment by majst01** [comment]
@fmachado should we first release 2.8 and postpone this into 2.9, what do you think ?

**Comment by fmachado** [comment]
IMO we could add it right now as it will only affect the existing PR (well, most of them must be fixed or discarded anyway). The other positive thing is that next PRs will already be created using junit5.

**Comment by majst01** [comment]
Agree, please merge it.

**Pull request merged**